### ISSUE TITLE
iOS: none translucent navigationBar gets translucent again on resume from background

### ISSUE THREAD

**Issue by Cookie8** [NativeScript/NativeScript#5904]
### Tell us about the problem
Hi,
For my iOS app I would like to have the navigationBar translucence off. 
It works fine, if the page will initially be loaded, but then, if I bring the app to background and bring it back to foreground again (resume), the navigationBar will be displayed 
translucent again.

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- CLI: 4.1.0 (but also occurred with CLI 4.0.2)
- Cross-platform modules: 4.2.0-2018-05-31-01
- Runtime(s): 
tns-android: 4.2.0-2018-05-31-01
tns-ios: 4.1.0
- Plugin(s): "dependencies": {
		"nativescript-iqkeyboardmanager": "^1.3.0",
		"nativescript-theme-core": "~1.0.4",
		"tns-core-modules": "^4.2.0-2018-05-31-01"
	},
	"devDependencies": {
		"babel-traverse": "6.26.0",
		"babel-types": "6.26.0",
		"babylon": "6.18.0",
		"lazy": "1.0.11"
	}

### Please tell us how to recreate the issue in as much detail as possible.
Please use the attached playground project below to reproduce this issue by following these steps:
1. Run the app -> The translucent is off and the NavigationBar looks as expected.
2. Bring the app in background by device home button (it must completely be in background)
3. Bring the app in foreground again -> The NavigationBar background looks to be translucent again, but as I can read in the logs the value of 'page.frame.ios.controller.navigationBar.translucent' will still be 'false'.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
Please see my Playground sample app:
[NSPlayground.zip](https://github.com/NativeScript/NativeScript/files/2064068/NSPlayground.zip)

The login page of the Playground sample project contains exactly what I implemented in my project and the issue also occurs in the Playground sample project.

Please just let me know, if you need further infos.
Cheers!

**Comment by NickIliev** [comment]
@Cookie8 there is a property called `flat` which you can use to turn of the translucency property.

For example,
```XML
<ActionBar title="Test" class="action-bar" flat="true">
</ActionBar>
```

And then there is no need to use the native property in the code behind
```JavaScript
const createViewModel = require("./login-view-model").createViewModel;

function loaded(args) {
    console.log("LOGIN-PAGE <-> loaded > start");
    const page = args.object;

    page.bindingContext = createViewModel();
    console.log("LOGIN-PAGE <-> loaded > end");
};
exports.loaded = loaded;

function navigatedTo(args) {
    console.log("LOGIN-PAGE <-> navigatedTo > start");
    const page = args.object;
    console.log("LOGIN-PAGE <-> loaded > end");
};
exports.navigatedTo = navigatedTo;
```

**Comment by Cookie8** [comment]
@NickIliev Thank you very much for your tip, however, if I use the flat property approach only, I will not have the vertical line on the bottom of the actionBar.

This is the result of using the flat property only:
<img width="375" alt="actionbar_using_property_flat" src="https://user-images.githubusercontent.com/2580190/40986666-11ba97f8-68e7-11e8-9721-59ce87afa00c.png">

And this is the result of using the translucent property:
<img width="375" alt="actionbar_using_property_translucent" src="https://user-images.githubusercontent.com/2580190/40986677-19d55496-68e7-11e8-9226-500ee6238a12.png">

Do you have any suggestion to handle the display of the actionBar's bottom line when using the flat property?

Thank you very much.

**Comment by NickIliev** [comment]
@Cookie8 the `flat` property is, in fact, removing the translucency, the "shadow" and the elevation and this is its main purpose to "flatten" the ActionBar.

**Comment by Cookie8** [comment]
@NickIliev Yes, I understand the purpose of the `flat` property and find it's very good to know it - Thank you again for the hint. 
But unfortunately in my case the 'flat' property won't be an alternative to the `translucent` property since I need the native look&feel of the ios navigationBar in my app. 
I can live with the current behaviour of the 'translucent' property for my further development, however, in my opinion it's a bug. Or am I wrong?
As I'm pretty new to the NativeScript community I'm not sure what to do now, to 'get the bug fixed' eventually. Do I need to report the issue somewhere else?

**Comment by NickIliev** [comment]
@Cookie8 thanks you for reporting this issue - here is the right place for posting a bug report. We will investigate the case and post any related information here

**Comment by ChaseXC** [comment]
I would like to note that I have run into an issue with the `flat` property as well, where all I need is the translucence set to false. Putting a SearchBar component within the ActionBar forces the ActionBar to grow a bit. When transitioning between pages, this growth is a smooth transition when `flat` is set to false, but looks choppy when `flat` is set to true. Trying to set translucent to false on the `loaded` event on the ActionBar for every page causes the ActionBar to switch back to translucent during the page transition, before switching again back to non-translucent.
